### PR TITLE
dev-libs/mdns: fix build on musl

### PR DESCRIPTION
**Pull request by DarthGandalf**
Signed-off-by: Alexey Sokolov <alexey+gentoo@asokolov.org>
Closes: https://bugs.gentoo.org/890790

### CODE REVIEW

**Comment by dlan17** [comment]
generally, the patch is trivial and looks good to me, but someone from @gentoo/musl team should take a look?

File: dev-libs/mdns/files/mdns-1.4.2-timeval.patch [hunk 1]
```diff
@@ -0,0 +1,12 @@
+https://github.com/mjansson/mdns/pull/69
+
+--- a/mdns.c
```
Comment (thesamesam) [inline dev-libs/mdns/files/mdns-1.4.2-timeval.patch:3]: Please send this upstream, but lgtm.

**Review by thesamesam — APPROVED** [review]

File: dev-libs/mdns/files/mdns-1.4.2-timeval.patch (hunk 1 above)
Comment (DarthGandalf) [reply dev-libs/mdns/files/mdns-1.4.2-timeval.patch:3]: Please see the comment on top of this file

Comment (thesamesam) [reply dev-libs/mdns/files/mdns-1.4.2-timeval.patch:3]: Sorry, not sure how I missed it!

**Pull request closed without merging**